src/main/java/com/unb/tracker/TrackerController.java

```diff
@@ -57,4 +60,37 @@ public String instructor(ModelMap map) {
         // This returns a JSON or XML with the users
         return userRepository.findAll();
     }
+
+    @GetMapping(value="/instructor/{courseId}") // courseId should be changed to name / section eventually
+    public String getCoursePage(@PathVariable Long courseId, ModelMap map) {
```

Comment:
The following method includes the actual page where instructors can edit their template, the rest are RESTful methods